.travis.yml

```diff
@@ -419,21 +419,40 @@ matrix:
 
 deploy:
   # See: https://docs.travis-ci.com/user/deployment/bintray/
-  provider: bintray
-  # NB: This is generated in after_success in each shard above.
-  file: ./native-engine.bintray.json
-  user: ${BINTRAY_USER}
-  key: ${BINTRAY_KEY}
-  dry-run: false
-  on:
-    condition: -f ./native-engine.bintray.json
-    # NB: Deploys are always tagged as part of the deploy process encoded in
-    # `build-support/bin/release.sh`, so this ensures we release an appropriate native engine binary
-    # for all releases. Unfortunately, CI only runs after the release tag hits origin and so there
-    # will be a lag of roughly 30 minutes until a pypi release has its paired native engine version
-    # available on bintray for OSX and linux. This trade-off (vs releasing for all branch builds),
-    # helps us use bintray in a friendly way.
-    tags: true
+  - provider: bintray
+    # NB: This is generated in after_success in each shard above.
+    file: ./native-engine.bintray.json
+    user: ${BINTRAY_USER}
+    key: ${BINTRAY_KEY}
+    dry-run: false
+    on:
+      condition: -f ./native-engine.bintray.json
+      # NB: Deploys are always tagged as part of the deploy process encoded in
+      # `build-support/bin/release.sh`, so this ensures we release an appropriate native engine binary
+      # for all releases. Unfortunately, CI only runs after the release tag hits origin and so there
+      # will be a lag of roughly 30 minutes until a pypi release has its paired native engine version
+      # available on bintray for OSX and linux. This trade-off (vs releasing for all branch builds),
+      # helps us use bintray in a friendly way.
+      tags: true
+      repo: pantsbuild/pants
+  # See: https://docs.travis-ci.com/user/deployment/s3/
+  - provider: s3
+    access_key_id: AKIAIQHTQI5E42SQBSNA
+    secret_access_key:
+      secure: RQVzsNfZL8AgsXdjZ67j2tWs5Tjl/FKpmE1fyVgldMbua/xhW8dzdFrtOeWjTPX4/+sJZ4U7/tZectBtWejmrXUJiZQKJwJBnsyYxysENTWOV80BEYyoz2RPr8HSVbMZ1ZHtUafzO3OqV1x+Pvgpg8FUeUfsy3TGUk0JREO90Q0=
```

Comment:
Its this: http://docs.travis-ci.com/user/encryption-keys/
We have 3 defined at the top of `.travis.yml` as well.